src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
```

Comment:
it's 100% be default

```suggestion
```

Reply:
add horizontal paddings of 50px to the header

Reply:
use flex to center the elements vertically and make the space between logo and nav (read about space-between value of justify-content)